streams/src/test/java/org/apache/kafka/streams/integration/NamedTopologyIntegrationTest.java

```diff
@@ -543,6 +543,16 @@ public void shouldAllowRemovingAndAddingNamedTopologyToRunningApplicationWithMul
 
     @Test
     public void shouldRemoveOneNamedTopologyWhileAnotherContinuesProcessing() throws Exception {
+        /*
+        Gradle Test Run :streams:unitTest > Gradle Test Executor 19 > NamedTopologyIntegrationTest > shouldRemoveOneNamedTopologyWhileAnotherContinuesProcessing() FAILED
+    java.lang.AssertionError:
+    Expected: <[KeyValue(B, 1), KeyValue(A, 2), KeyValue(C, 2)]>
+         but: was <[KeyValue(B, 1), KeyValue(A, 2), KeyValue(C, 1)]>
+        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
+        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
+        at org.apache.kafka.streams.integration.NamedTopologyIntegrationTest.shouldRemoveOneNamedTopologyWhileAnotherContinuesProcessing(NamedTopologyIntegrationTest.java:563)
+
+         */
```

Comment:
is there an issue with this test? It passes for me locally on your PR and on trunk.

Reply:
Ups. This should not go into this PR. Good catch.

The test is flaky and it was not to myself for my local copy only.